src/dodal/devices/i04/beam_centre.py

```diff
@@ -0,0 +1,72 @@
+import cv2
+import numpy as np
+from bluesky.protocols import Triggerable
+from ophyd_async.core import AsyncStatus, StandardReadable, soft_signal_r_and_setter
+from ophyd_async.epics.core import (
+    epics_signal_r,
+)
+
+from dodal.devices.i04.max_pixel import convert_to_gray_and_blur
+from dodal.log import LOGGER
+
+
+async def binary_img(img, img_name="Threshold"):
```

Comment:
Should: Give types for all our parameters and return values


```suggestion
async def binary_img(img: cv2.typing.MatLike, img_name: str = "Threshold"):
```